src/util/scheduler.ts

```diff
@@ -0,0 +1,29 @@
+import Container from "typedi";
+import { CloudantService, FlashCardService } from "../services";
+
+var CronJob = require("cron").CronJob;
+
+var flashCardService: FlashCardService = Container.get("FlashCardService");
```

Comment:
How come you are using ES5 here? Should change this to ES6 syntax to remain consistent

Reply:
good spot I didnt realise this :)